lib/widgets/auth/auth_page.dart

```diff
@@ -159,69 +161,78 @@ class PlatformSignInButton extends StatelessWidget {
   }
 }
 
-class OtherOptionsButton extends StatelessWidget {
+class OtherDefaultProviderButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return ButtonTheme(
-      height: 40.0,
-      padding: EdgeInsets.only(left: 30.0, right: 30.0),
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(3.0),
-        side: BorderSide(
-          color: Colors.black,
-        ),
-      ),
-      child: RaisedButton(
-        onPressed: () =>
-            context.dispatch(PushPage(data: OtherAuthOptionsPageData())),
-        color: Colors.white,
-        child: Row(
-          mainAxisSize: MainAxisSize.min,
-          children: [
-            Text(
-              'More Options',
-              style: TextStyle(
-                  fontSize: 16.0,
-                  fontFamily: 'SF Pro',
-                  fontWeight: FontWeight.w500,
-                  color: Colors.black),
-            ),
-          ],
-        ),
-      ),
-    );
+    return StoreConnector<AppState, PlatformType>(
+        distinct: true,
+        converter: (store) => store.state.systemInfo.platform,
+        builder: (context, platform) {
+          return (platform == PlatformType.ios ||
+                  platform == PlatformType.macOS)
+              ? GoogleSignInFAB()
+              : AppleSignInFAB();
+        });
+  }
+}
+
+class GoogleSignInFAB extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return FloatingActionButton(
+        child: ImageIcon(AssetImage('assets/images/google_logo.png')),
+        elevation: 1,
+        mini: true,
+        onPressed: () => context.dispatch(SignInWithGoogle()));
+  }
+}
+
+class AppleSignInFAB extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return FloatingActionButton(
+        child: ImageIcon(AssetImage('assets/images/apple_logo_black.png')),
+        elevation: 1,
+        mini: true,
+        onPressed: () => context.dispatch(SignInWithApple()));
   }
 }
 
 class EmailOptionsButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return ButtonTheme(
```

Comment:
just a normal button to a FAB, same behaviour

Reply:
sweet